[WIP] Catch deprecation warnings of random_state in svc tests

FarahSaeed opened the pull request:
#### Reference Issues/PRs
Reference Issue: #10158


Hi.. Regarding issue #10158, this PR is related to catching deprecation warnings of random_state in SVC tests.
I've found the following tests where the relevant deprecation warnings are raising.
https://github.com/scikit-learn/scikit-learn/blob/57bcd075c39d8bd45c4b9b2c1378d2010f467cd1/sklearn/svm/tests/test_sparse.py#L264

In this PR, the above test is edited to catch deprecation warning.

As for now, I've kept this PR '[WIP]' as there is need to make sure wether random_state is not being used in \examples or \doc.

FarahSaeed committed:
Catch deprecation warnings of random_state in svc

Catch deprecation warning of svc random_state in tests.

jnothman commented on sklearn/svm/tests/test_sparse.py:
```diff
@@ -273,9 +273,10 @@ def test_sparse_oneclasssvm():
-            sp_clf = svm.OneClassSVM(kernel=kernel, random_state=0)
-            check_svm_model_equal(clf, sp_clf, *dataset)
+            with ignore_warnings(category=DeprecationWarning):		
+                clf = svm.OneClassSVM(kernel=kernel, random_state=0)
```
nah. Just remove the use of `random_state` instead. It's never done anything, and the deprecation is checked elsewhere.

jnothman reviewed (commented):
Thanks!

FarahSaeed committed:
Removing 'random_state = 0'  while intializing OneClassSVM object

Removing 'random_state = 0'  while intializing OneClassSVM object. (This
doesn't have any effect on functionality)

FarahSaeed commented:
@jnothman Thanks for the advice. In recent commit, I've removed the random_state parameter. I would also look for its use in /doc and /examples.

FarahSaeed commented:
Let me open a new pull request. In this one, there are a lot of other changes as well which I did for syncing.
